index.html

```diff
@@ -2,6 +2,7 @@
 <html>
     <head>
         <title>Git workflow experiment</title>
+        <link rel="stylesheet" type="text/css" href="main.css">
```

Comment:
I like that we are using `link` and not `@import`, which is another way to include CSS from an external file.